test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1060,6 +1060,83 @@ def validate(self, model: torch.fx.GraphModule) -> None:
 
         self._test_transitive_sharing_with_cat_helper(BackendAQuantizer())
 
+    def test_allow_implicit_sharing(self):
+        """This tests the allow_transitive_sharing flag of QuantizationAnnotation, that is
+        if a node is configured with allow_implicit_sharing=False, we will not have implicit sharing
+        for node and (node, consumer) even they refer to the same Tensor
+
+        x1 -> add1 -----> add3
+        x2 -/              /
+               x3 -> add2 /
+               x4 -/
+
+        all add has shared input and output, and second input is using shared quantization spec pointint
+        to first input, but we set allow_implicit_sharing to all add nodes so input and output of add1,
```

Comment:
you mean set allow_implicit_sharing to false?

Reply:
yeah, will fix